src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -206,13 +228,16 @@ public static LocalDate parseDate(String date) throws ParseException {
         String dateValidation = "\\d{2}-\\d{2}-\\d{4}";
         DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("dd-MM-yyyy");
```

Comment:
Same for this DateTimeFormatter and in the `parseValidAppointmentDate()`, with another existing regex in the `Appointment` class. (Unhinged) Maybe we could even make a static Date class to hold all this functionality?

Reply:
@Kb-Tay I'd like your input on these too, since you're the original author of this code.